portfolio.html

```diff
@@ -27,23 +28,56 @@ <h2>About me</h2>
 			  <img src="1516887791047.jpeg" alt="An image of Grace Palacz"> 
 		    </div>
 			  <p>Hi I'm Grace, a full stack software developer based near Bristol, UK.
-			  Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum. </p> 
+			  Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum.</p> 
 		  </section>
-			<div class="projectsIntro">
-			  <h2>Projects</h2>
+			<div id="portfolio" class="projectsIntro">
+			  <h2>Portfolio</h2>
 		    </div>	
 		      <main>
-				<div class="projectOne projectBox">
-				    </div>
-				    <div class="projectTwo projectBox">
-				    </div>
-				    <div class="projectThree projectBox">
-				    </div>	  	  
+				<div class="project projectBox">
+					<h3>Project title</h3>
+						<img src="Header.jpg" alt="Placeholder Image" />
+						<p>Some text about the project goes here. Blah blah blah... Even more description...</p>
+						  <div class="projectLinks">
+						    <a href="#" target="_blank">Github
+						      <i class="fab fa-github"></i>
+					        </a>
+   						      <a href="#" target="_blank">Live
+   						        <i class="fas fa-arrow-circle-right"></i>
+   					          </a>
+					      </div>  
+				</div>
+				  <div class="project projectBox">
+					<h3>Project title</h3>
+						<img src="Header.jpg" alt="Placeholder Image" />
+						<p>Some text about the project goes here. Blah blah blah...Even more description...</p>
+						  <div class="projectLinks">
+						    <a href="#" target="_blank">Github
+						      <i class="fab fa-github"></i>
+					        </a>
+   						      <a href="#" target="_blank">Live
+   						        <i class="fas fa-arrow-circle-right"></i>
+   					          </a>
+					      </div>
+				  </div>
+				    <div class="project projectBox">
+						<h3>Project title</h3>
+							<img src="Header.jpg" alt="Placeholder Image" />
+						    <p>Some text about the project goes here. Blah blah blah... Even more description...</p>
```

Comment:
Could the indentation of each line on HTML file be adjusted to show a clearer child/parent relationship between elements?